health/micro-ui-react19/web/packages/css/src/pages/employee/healthdss.scss

```
@@ -974,6 +979,21 @@
   margin-bottom: 1.5rem;
   display: flex;
   gap: 2rem;
+
+  width: 80%;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)